TrackletAlgorithm/MatchEngineUnit_parameters.h

```diff
@@ -80,6 +99,22 @@ static bool nearFull4UnitBool(ap_uint<kNBitsBuffer> rptr, ap_uint<kNBitsBuffer>
   return wptr1==rptr || wptr2==rptr || wptr3==rptr || wptr4==rptr;
 }
 
+template<int kNBitsBuffer>
+class emptyUnitClass {
```

Comment:
CMS coding rules state that class names should start with a capital letter, whereas variable and function names should start with a lower case letter.